src/akn/Metadata.cs

```diff
@@ -231,11 +234,21 @@ public static XmlElement make(XmlDocument doc, IJudgment judgment, IMetadata met
                 proprietary.AppendChild(cite);
                 cite.AppendChild(doc.CreateTextNode(metadata.Cite.ToString()));
             }
+
+
+            foreach (var caseNo in metadata.CaseNos()) {
+                MetadataExtensions.AddProprietaryField(proprietary, "caseNumber", caseNo);
```

Comment:
why do we add these with MetadataExtensions but the other proprietaryfields just added with `proprietary.AppendChild(juris);` or `proprietary.AppendChild(cite);`